### PR TITLE
feat: Auto mute on accepting incoming conference call

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: src/script/calling/CallingRepository.ts [hunk 1]
```diff
@@ -664,8 +664,9 @@ export class CallingRepository {
     }
   };
 
-  async answerCall(call: Call, callType: CALL_TYPE): Promise<void> {
+  async answerCall(call: Call, callType?: CALL_TYPE): Promise<void> {
     try {
+      callType = callType ?? call.getSelfParticipant().sharesCamera() ? call.initialType : CALL_TYPE.NORMAL;
```
Comment (ffflorian) [inline src/script/calling/CallingRepository.ts]:
```suggestion
      callType ??= call.getSelfParticipant().sharesCamera() ? call.initialType : CALL_TYPE.NORMAL;
```

Comment (lipis) [reply src/script/calling/CallingRepository.ts]: never seen this syntax.. interesting!!

Comment (ffflorian) [reply src/script/calling/CallingRepository.ts]: See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html :slightly_smiling_face:

**Review by thisisamir98 — APPROVED** [review]

**Comment by thisisamir98** [comment]
LGTM!

**Pull request merged**